content/app/development/configuration/authorisation/_index.en.md

```diff
@@ -3,43 +3,42 @@ title: Authorization
+The rules that are defined in this file is described below.  
 
-Overnevnte regler kan endres i *policy.xml* som ligger i `App/config/authorization` i applikasjonsrepoet.
-Detaljer om hvordan å konfigurere policy filen finnes [her](/technology/architecture/components/application/solution/altinn-studio/designer/pap/xacmlpolicy/)
-I tillegg vil du finne en del eksempelregler [her](rules).
+The forementioned rules can be changed in *policy.xml* which lies in `App/config/authorization` in the application repository.
+Detailes about configuring the policy file can be found [here](/technology/architecture/components/application/solution/altinn-studio/designer/pap/xacmlpolicy/
+You can also find a good amount of example rules [here](rules).
 
 {{%notice warning%}}
-Merk at endringer i policyfilen gjøres på eget ansvar, 
-og at det oppfordres til å alltid delegere leserettigheter dersom en entitet også har fått tildelt skriverettigheter.
+Please note that changes to the policy file is at your own risk,
+and that it is recommended to always delegate read permissions to entities with writing permissions.
 {{% /notice%}}
 
-### Rettigheter for rolleinnhaver
-I denne filen er det definert regler som gir innehaver av rollene daglig leder (DAGL) og/eller regnskapsmedarbeider (REGNA)
-rettigheter til å instansiere, skrive, lese og slette instanser av applikasjonen.
+### Rights for role haver
+The rules that give the daily leader (DAGL) or Accountant assistant (REGNA) permissions to instantiate, write 
+read and delete instances of the applicationare defined in the policy file.
 
-En fullstendig liste over rolletyper finner du [her](https://www.altinn.no/api/metadata/roledefinitions) .
+A complete list of role types can be found [here](https://www.altinn.no/api/metadata/roledefinitions).
 
-### Rettigheter for applikasjonseier
-Applikasjonseier (organisasjonen) har rettigheter til å instansiere, skrive og lese instanser av applikasjonen.
-Til slutt har de rettigheter til å markere på en instans at de er ferdig med den.
+### Rights for the application owner
+The application owner (organization) has rights to instantiate, write and read instances of the application.
+They also have permissions to mark instances as completed. 
 
-### Påkrevd autentiseringsnivå
-Påkrevd autentiseringsnivå er satt til 2 som default. Dette gjøres som en obligation i XACML Policy
+### Required authentication level
+Reqired authentication level is set to 2 by default. This is done as an obligation in the XACML Policy file. 
 
-Hvis nivået settes til 4 må man definere at tjenesteier kan nå det via nivå 3 for maskinporten. Dette for at maskinporten er definert som nivå 3. 
-Se regelbibliotek for eksempel. Merk: Appen trenger nuget versjon `3.1.5` eller høyere versjon.
+If the required authentication level is set to 4 you have to define that the service owner can interact with it through level 3 authentification for Maskinporten.
```

Comment:
```suggestion
If the required authentication level is set to 4 you have to define that the service owner can interact with it through level 3 authentication for Maskinporten.
```
I looked it up, and it turns out 'authentification' is valid, but less commonly used than simpliy 'authentication': https://english.stackexchange.com/a/7845